src/containers/NFT/NFTHeader/index.js

```diff
@@ -0,0 +1,246 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { withTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { loadNFTState } from './actions';
+import Loader from '../../shared/components/Loader';
+import '../../shared/css/nested-menu.css';
+import './styles.css';
+import SocketContext from '../../shared/SocketContext';
+import Tooltip from '../../shared/components/Tooltip';
+import Copy from '../../shared/components/Copy';
+
+class NFTHeader extends Component {
+  constructor(props) {
```

Comment:
we're transitioning away from class-based components and to functional components instead (IE hooks).

Reply:
will change